tensorboard/components/security.html

```diff
@@ -0,0 +1,37 @@
+<!--
+@license
+Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<script>
+  window.Polymer = {strictTemplatePolicy: true};
+</script>
+<!-- These polymer iron/paper components violate the strictTemplatePolicy. -->
```

Comment:
Can you just briefly elaborate to explain why the lines below address the issue?  Aka "... violate the strictTemplatePolicy by not declaring a <dom-module> element or template.  We define them here ourselves so that strictTemplatePolicy will be satisfied."